docs-requirements.txt

```diff
@@ -6,7 +6,7 @@ babel==2.9.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2,
 certifi==2021.5.30
 chardet==4.0.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 click-default-group==1.2.2
-click==8.0.1; python_version >= '3.6'
+click==7.1.2; python_version >= '3.6'
```

Comment:
Curious - what made you rollback the click version here?

Reply:
To avoid conflict with `requirements.txt`. Come to think of it, these dependencies should never conflict because we treat them as separate environments. WDYT?

Reply:
If the docs requirements have a different version, conflicts can occur for local development and for circle ci builds.

If I run `pip install -e . -r docs-requirements.txt` locally I get the following:
```
ERROR: Cannot install click==8.0.1 and nucypher==5.3.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested click==8.0.1
    nucypher 5.3.0 depends on click==7.1.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

We've probably gotten away with this mismatch because of circle ci caching. It's best for the requirements to match. If updating one, then update the other, which I thought we did in the `relock_dependencies.sh` script...

The update of the docs-requirements to use the newer version of click seemed to occur in https://github.com/nucypher/nucypher/commit/b91dd386b849c748c077b4bfe34fdbeccb6b38cd as part of #2612 .

Reply:
Probably need to do a relock of dependencies, and increment the circle ci cache.